topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md

```diff
@@ -0,0 +1,744 @@
+> - The workflow for ONT-sequenced ARTIC data (ONT ARTIC) is modeled after the alignment/AV-calling steps of the [ARTIC pipeline](https://artic.readthedocs.io/). It performs, essentially, the same steps as that pipeline’s minion command, i.e. read mapping with **minimap2** and AV calling with **medaka**. Like the Illumina ARTIC workflow it uses **ivar** for primer trimming. Since ONT-sequenced reads have a much higher error rate than Illumina-sequenced reads and are therefore plagued more by false-positive AV calls, this workflow makes no attempt to handle amplicons affected by potential primer-binding site mutations.
+>
+> All four workflows use **SnpEff**, specifically its 4.5covid19 version, for AV annotation.
+>
+> Workflows default to requiring an AF ≥ 0.05 and AV-supporting reads of ≥ 10 (these and all other parameters can be easily changed by the user). For an AV to be listed in the reports it must surpass these thresholds in at least one sample of the respective dataset. We estimate that for AV calls with an AF ≥ 0.05 our analyses have a false-positive rate of < 15% for both Illumina RNAseq and Illumina ARTIC data, while the true-positive rate of calling such low-frequency AVs is ~80% and approaches 100% for AVs with an AF ≥ 0.15. This estimate is based on an initial application of the Illumina RNAseq and Illumina ARTIC workflows to two samples for which data of both types had been obtained at the virology department of the University of Freiburg and the assumption that AVs supported by both sets of sequencing data are true AVs. The second threshold of 10 AV-supporting reads is applied to ensure that calculated AFs are sufficiently precise for all AVs.
+>
+> More details about the workflows, including benchmarking of the tools, can be found on [covid19.galaxyproject.org](https://covid19.galaxyproject.org/genomics/global_platform/#methods)
+{: .details}
+
+> ### {% icon hands_on %} Hands-on: From FASTQ to annotated AVs
+>
+> 1. **Get the workflow** on Galaxy 
+>
+>    - Option 1: Use workflows directly on [usegalaxy.eu](https://usegalaxy.eu/) from [WorkflowHub](https://workflowhub.eu)
+>      - Open the workflow page on WokflowHub
+>        - [Illumina ARTIC PE](https://workflowhub.eu/workflows/110) - The one to use for example datasets
+>        - [Illumina RNAseq SE](https://workflowhub.eu/workflows/112)
+>        - [Illumina RNAseq PE](https://workflowhub.eu/workflows/113)
+>        - [ONT ARTIC](https://workflowhub.eu/workflows/111)
+>      - Click on `Run on usegalaxy.eu` on the top right of the page
+>      
+>        The browser will open a new tab with Galaxy's workflow invocation interface.
+>
+>    - Option 2: Import the workflows of IWC (Intergalactic Workflow Commission) from WorkflowHub using the workflow search
```

Comment:
Maybe provide instructions here for import from dockstore cause that will work on usegalaxy.org as well.

Reply:
This has the additional advantage that the sixth (the consensus) WF is available there.